src/rest/response.cpp

```diff
@@ -49,7 +48,7 @@ Response::Response(void)
     mProtocol = "HTTP/1.1";
 
     // Pre-defined headers
-    mHeaders["Content-Type"]                 = OT_REST_RESPONSE_CONTENT_TYPE_JSON;
+    mHeaders[OT_REST_CONTENT_TYPE_HEADER]    = OT_REST_CONTENT_TYPE_JSON;
     mHeaders["Access-Control-Allow-Origin"]  = OT_REST_RESPONSE_ACCESS_CONTROL_ALLOW_ORIGIN;
     mHeaders["Access-Control-Allow-Methods"] = OT_REST_RESPONSE_ACCESS_CONTROL_ALLOW_METHOD;
```

Comment:
Not all resources support these `PUT`, should we set this by resource?

Reply:
It doesn't matter much as this is only about which methods are allowed. Since they are not implemented, it doesn't really matter weather they are allowed.